pkg/registry/core/service/ipallocator/metrics.go

```diff
@@ -60,9 +60,9 @@ var (
 			Help:           "Number of Cluster IPs allocations",
 			StabilityLevel: metrics.ALPHA,
 		},
-		[]string{"cidr"},
+		[]string{"cidr", "allocation_mode"},
```

Comment:
"scope" would be shorter?  Is it much worse?

Reply:
scope sounds good